### ISSUE TITLE
Failure to skull strip / appropriately resize image 

### ISSUE THREAD

**Issue by kellyhi08** [DCAN-Labs/BIBSnet#97]
### What happened?

I have been running BIBSnet on a group of infants and for a few it is segmenting non-brain (face, spine, mouth). These subjects do not have qualitatively worse (higher motion) T1s than some of my other subjects, so I am not sure what is leading to this issue. All subject T1s were collected with the same scan parameters. My subjects range in age from 1 month to 2 years. The three problem subjects are ages 1 month, 11 moths and 3 months. BIBSnet did work on one of my 2 month subjects and multiple 1 year old subjects. Initially, I did not include brain size, but when this problem arose, I tried adding it, but that did not help. 

Here are some images of an example failing subject: 
**T1**
![image](https://github.com/DCAN-Labs/BIBSnet/assets/63243958/c26d14ad-5c2e-4908-8053-895d37704380)

**Failing segmentation**
![image](https://github.com/DCAN-Labs/BIBSnet/assets/63243958/c1e80b4a-2488-41ad-b40c-d6e42b10aded)

### What command did you use?

```shell
singularity pull bibsnet.sif docker://dcanumn/bibsnet:latest 

singularity run --nv --cleanenv --no-home -B path/BIBSnet_inputs/:/input -B path/BIBSnet_outputs:/output path/BIBSnet/bibsnet.sif /input /output participant -v --participant-label $subnum -z
```

### What version of BIBSnet are you using?

latest as of Jan 16th, 2024 

### Directory Structure

There are more subjects, but this is the start of the tree. All subjects look the same. 

![image](https://github.com/DCAN-Labs/BIBSnet/assets/63243958/8f86f5ea-c4a6-4afe-b9ec-bacb1b5d0e79)

**Comment by LuciMoore** [comment]
@kellyhi08 can you try to run fslreorient2std or fslorient (see documentation [here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Orientation%20Explained)) on the raw data and see if this fixes your issue? I think this could also be causing the other comment you made [here](https://github.com/DCAN-Labs/BIBSnet/issues/84#issuecomment-1928101537) 

If one of these works then we will add this command to the preBIBSNet stage of processing to prevent future issues

**Comment by kellyhi08** [comment]
Hello,

I tried fslerorient2std, but that did not seem to make a difference. However, the images were oriented incorrectly in FSL, so I tried a different dicom to nifti converter (dicm2nii in matlab), and that fixed the orientation and that seemed to fix the segmentation problem! 

![image](https://github.com/DCAN-Labs/BIBSnet/assets/63243958/7c7396ab-9177-462a-942e-d7d5c3c63e6f)


Thanks for your help!
